### PR TITLE
Added some javadoc and optimized code

### PR DESCRIPTION
**Pull request by triszt4n**
Unfortunately the code formatting made some stupid whitespace corrections, sorry, on review page, there is an option for omitting whitespace modifications

### CODE REVIEW

File: java/src/hu/holyoil/controller/TurnController.java [hunk 1]
```diff
@@ -18,14 +18,19 @@
...
     /**
      * Hány játékos lehet maximum egy játékban
      */
     public final int NUM_OF_PLAYERS_MAX = 4;
 
+    /**
+     * Hány játékos lehet minimum egy játékban
+     */
+    public final int NUM_OF_PLAYERS_MIN = 2;
+
```
Comment (triszt4n) [inline java/src/hu/holyoil/controller/TurnController.java:33]: This is new

File: java/src/hu/holyoil/view/frames/GameFrame.java [hunk 2]
```diff
@@ -11,19 +11,22 @@
...
 
     private void InitComponent() {
-        logPanel = new LogPanel();
-        environmentPanel = new EnvironmentPanel();
-        inventoryListPanel = new InventoryListPanel();
-        playerListPanel = new PlayerListPanel();
+        LogPanel logPanel = new LogPanel();
+        EnvironmentPanel environmentPanel = new EnvironmentPanel();
+        InventoryListPanel inventoryListPanel = new InventoryListPanel();
+        PlayerListPanel playerListPanel = new PlayerListPanel();
```
Comment (triszt4n) [inline java/src/hu/holyoil/view/frames/GameFrame.java:29]: Made them local

File: java/src/hu/holyoil/view/panels/EnvironmentPanel.java [hunk 3]
```diff
@@ -34,21 +39,27 @@
     private JLabel sunstormCountLabel;
 
     // Képek
-    private final Image asteroidImg = new ImageIcon("assets/plain_asteroid.png").getImage();
-    private final Image darkImg = new ImageIcon("assets/plain_asteroid_dark.png").getImage();
-    private final Image teleportImg = new ImageIcon("assets/teleporter.gif").getImage();
+    private static final Image asteroidImg = new ImageIcon("assets/plain_asteroid.png").getImage();
+    private static final Image darkImg = new ImageIcon("assets/plain_asteroid_dark.png").getImage();
+    private static final Image teleportImg = new ImageIcon("assets/teleporter.gif").getImage();
```
Comment (triszt4n) [inline java/src/hu/holyoil/view/panels/EnvironmentPanel.java:44]: Made them static

File: java/src/hu/holyoil/view/panels/LogPanel.java [hunk 4]
```diff
@@ -37,7 +48,7 @@ private void InitComponent() {
         giveUpButton.setPreferredSize(new Dimension(140, 50));
 
         jTextArea = new JTextArea(35, 50);
-        jScrollPane = new JScrollPane(jTextArea);
+        JScrollPane jScrollPane = new JScrollPane(jTextArea);
```
Comment (triszt4n) [inline java/src/hu/holyoil/view/panels/LogPanel.java:51]: Made it local

**Review by captEmma — APPROVED** [review]
ty for ur services

**Pull request merged**